From 8fb63b4e18c38372dccfe69c28d1241c6c4e6607 Mon Sep 17 00:00:00 2001
From: J. Bruce Fields <bfields@redhat.com>
Date: Tue, 16 Sep 2014 20:20:08 -0400
Subject: [fs] nfsd: rename unhash_generic_stateid to unhash_ol_stateid

Message-id: <1410898849-22126-125-git-send-email-bfields@redhat.com>
Patchwork-id: 95731
O-Subject: [Patch RHEL7 124/165] nfsd: rename unhash_generic_stateid to unhash_ol_stateid
Bugzilla: 1044798 1123721 1141932
RH-Acked-by: Benjamin Coddington <bcodding@redhat.com>
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Jeff Layton <jlayton@primarydata.com>

...to better match other functions that deal with open/lock stateids.

Signed-off-by: Jeff Layton <jlayton@primarydata.com>
Signed-off-by: J. Bruce Fields <bfields@redhat.com>
Upstream: 4ae098d327c599c9a8e2eecedcc2c192b537ff4e
Bugzilla: 1044798
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfsd/nfs4state.c b/fs/nfsd/nfs4state.c
index 1f67a96..52ec47d 100644
--- a/fs/nfsd/nfs4state.c
+++ b/fs/nfsd/nfs4state.c
@@ -949,7 +949,7 @@ static void nfs4_put_stateowner(struct nfs4_stateowner *sop)
  sop->so_ops->so_free(sop);
 }
 
-static void unhash_generic_stateid(struct nfs4_ol_stateid *stp)
+static void unhash_ol_stateid(struct nfs4_ol_stateid *stp)
 {
  struct nfs4_file *fp = stp->st_stid.sc_file;
 
@@ -1014,7 +1014,7 @@ static void unhash_lock_stateid(struct nfs4_ol_stateid *stp)
  lockdep_assert_held(&oo->oo_owner.so_client->cl_lock);
 
  list_del_init(&stp->st_locks);
- unhash_generic_stateid(stp);
+ unhash_ol_stateid(stp);
  unhash_stid(&stp->st_stid);
 }
 
@@ -1095,7 +1095,7 @@ static void unhash_open_stateid(struct nfs4_ol_stateid *stp,
 {
  lockdep_assert_held(&stp->st_stid.sc_client->cl_lock);
 
- unhash_generic_stateid(stp);
+ unhash_ol_stateid(stp);
  release_open_stateid_locks(stp, reaplist);
 }
 
-- 
1.7.1